web/src/engine/transformers/ImageLinkDeProxifier_test.ts

```diff
@@ -18,11 +18,13 @@ describe('ImageProxyTransformer', () => {
 
         it('Should transform photon proxy', () => {
             const uri = new URL('https://i0.wp.com/cdn.image.host/picture.png');
-            expect(testee(uri).href).toBe('http://cdn.image.host/picture.png');
+            expect(testee(uri).href).toBe('https://cdn.image.host/picture.png');
             uri.searchParams.set('ssl', '1');
             expect(testee(uri).href).toBe('https://cdn.image.host/picture.png');
+            uri.searchParams.set('ssl', '0');
+            expect(testee(uri).href).toBe('http://cdn.image.host/picture.png');
             uri.searchParams.set('q', 'token=123&expire=987');
```

Comment:
Also remove `ssl` parameter before last assertion so a broader range of permutations is tested

Reply:
I did that then i changed my mind lmao :D